tests/GoogleAuthenticatorTest.php

```diff
@@ -120,4 +120,38 @@ public function testGetUrlNoIssuer(): void
             $this->helper->getUrl('foo', 'foobar.org', '3DHTQX4GCRKHGS55CJ')
         );
     }
+
+    public function testPeriodValid(): void
```

Comment:
An you use a dataProvider for the cases?

Reply:
Am I what? ;-)

I presumed that the test cases were functions prefixed with "test" that were automatically picked up.